### ISSUE TITLE
Cleanup entries messes up single quotes

### ISSUE THREAD

**Issue by crystalfp** [JabRef/jabref#3140]
JabRef 3.8.2
windows 7 6.1 amd64 
Java 1.8.0_144

Load the attached entry. Note the word between single quotes in the abstract. Now select the entry and run cleanup entry with the settings of the attached screenshot.

![bug-single-quote](https://user-images.githubusercontent.com/865866/29493960-ca83990e-85a0-11e7-95a1-1bdbe1431f03.png)

you see that " to a 'gap' ́between" become "to a ǵap ́between"

If I change the entry to follow latex rules for single quotes, then "to a `gap' between" becomes "to a `gap ́between".

The simple workaround is to use double quotes that are not changed by the cleanup or use something different from "HTML to LaTeX" in the setup dialog. Note that the entry has been copied verbatim from the web. Thanks!

  ```
  @Article{Howard-Jones2014,
  author       = {Howard-Jones, Paul A.},
  title        = {Neuroscience and education: myths and messages},
  journaltitle = {Nat Rev Neurosci},
  date         = {2014-10-15},
  volume       = {15},
  number       = {12},
  pages        = {817--824},
  issn         = {1471-003X},
  doi          = {10.1038/nrn3817},
  url          = {http://www.nature.com/nrn/journal/v15/n12/full/nrn3817.html},
  abstract     = {For several decades, myths about the brain --- neuromyths --- have persisted in schools and colleges, often being used to justify ineffective approaches to teaching. Many of these myths are biased distortions of scientific fact. Cultural conditions, such as differences in terminology and language, have contributed to a 'gap' between neuroscience and education that has shielded these distortions from scrutiny. In recent years, scientific communications across this gap have increased, although the messages are often distorted by the same conditions and biases as those responsible for neuromyths. In the future, the establishment of a new field of inquiry that is dedicated to bridging neuroscience and education may help to inform and to improve these communications.},
  langid       = {english},
}
  ```

**Comment by lenhard** [comment]
@crystalfp Thanks for your report. This is a bug in the LaTeX conversion function that should already be fixed in the newer version. Could you please try again with the most recent build from http://builds.jabref.org/master/ and report back if it works?

**Comment by crystalfp** [comment]
OK solved under `JabRef-4.0-beta3.jar`. Thanks!
